addons/play-json/README.md

```diff
@@ -0,0 +1,49 @@
+handlebars-play-json
+====================
+
+Play-JSON data bindings for handlebars.scala
```

Comment:
May be worth noting that it only works for play 2.4.x. I know at Gilt we still use 2.3.x quite a bit and I am not sure we'd be able to use this (which is fine, because we should upgrade). Just wanted to see it documented!